crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -145,65 +133,59 @@ impl IndexerReader {
         .expect("propagate any panics")
     }
 
-    pub fn get_pool(&self) -> ConnectionPool {
-        self.pool.clone()
+    pub fn get_pool(&self) -> BlockingConnectionPool {
+        self.blocking_pool.clone()
     }
 }
```

Comment:
is this actually still used anywhere? seems like we just do `self.pool.get().await` now?

Reply:
its used in graphql to get connections, in a follow up PR i'll work to convert graphql codebase which will then let us remove the blocking pool entirely